### PR TITLE
Update of Ukrainian Translation

### PR DESCRIPTION
**Pull request by ZaHaDumGoth**
Mostly hotkey and some empty rows founded.

### CODE REVIEW

File: files/lang/uk.po [hunk 1]
```diff
@@ -7388,25 +7419,35 @@ msgstr "Згуртувати війська"
...
+"Встановити портрет героя. Клацніть правою кнопкою миші, щоб відновити "
+"значення за замовчуванням."
 
 msgid "Click to change hero's name. Right-click to reset to default."
 msgstr ""
+"Тицьніть для зміни імені героя. Тицьніть правою кнопкою миші, щоб відновити "
+"значення за замовчуванням."
 
 msgid "Hero is in patrol mode in %{tiles} tiles radius. Click to disable it."
 msgstr ""
```
Comment (ihhub) [inline files/lang/uk.po:7445]: {tiles} must not be translated.

**Review by ihhub — CHANGES_REQUESTED** [review]
Hi @ZaHaDumGoth , I left one comment here. Could you please take a look?

File: files/lang/uk.po (hunk 1 above)
Comment (ZaHaDumGoth) [reply files/lang/uk.po:7445]: Fixed it.

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@ZaHaDumGoth , many thanks for this update!

**Pull request merged**